servers/physics_3d/godot_body_3d.h

```diff
@@ -376,7 +381,7 @@ void GodotBody3D::add_contact(const Vector3 &p_local_pos, const Vector3 &p_local
 			idx = least_deep;
 		}
 		if (idx == -1) {
-			return; //none least deepe than this
+			return -1; //none least deepe than this
```

Comment:
Is this supposed to be something akin to:
> No contacts less deep than this.

deepe seems like a typo.

Reply:
It's a pun, it a literature reference

Reply:
Haha I'm not sure it is, it might just be a typo :P 
I don't find much references to "least deepe" or just "deepe" in literature.

Here it does refer back to the `least_deep` variable though.